addons/mail/static/src/core/web/discuss_core_common_service_patch.js

```diff
@@ -0,0 +1,34 @@
+import { DiscussCoreCommon } from "@mail/discuss/core/common/discuss_core_common_service";
+import { patch } from "@web/core/utils/patch";
+
+patch(DiscussCoreCommon.prototype, {
+    _handleNotificationChannelDelete(thread, metadata) {
```

Comment:
something defined in discuss/ subfolder should only be referenced or patched in another discuss/ subfolder